src/app/states/facade/usersFacade.ts

```diff
@@ -0,0 +1,28 @@
+import {Injectable} from '@angular/core';
+import {Store} from '@ngxs/store';
+import {Observable} from 'rxjs';
+import {User} from '../../interfaces/user';
+import {AddUser, AddUsers} from '../actions/user.action';
+
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UsersFacade {
+  constructor(
+    private store: Store
+  ) {}
+
+  getAllUsers(): Observable<User[]> {
+    return this.store.select( state => state.user.users);
```

Comment:
Bestenfalls einen memoized Selector verwenden (https://www.ngxs.io/concepts/select#memoized-selectors):
```suggestion
    return this.store.select(UserState.users);
```